README.md

```diff
@@ -20,136 +20,166 @@ Or [npm](https://www.npmjs.com/):
 npm install @zaiusinc/node-sdk
 ```
 
-## Configuration
+## Usage
+
+To communicate with ODP, you need to obtain and configure an instance of `ODPClient`. 
+There are two ways to do this:
+1. Use module scoped instance exported as `odp` from the SDK.
+2. Create your own instance of `ODPClient`.
+
+Using module scoped instance is easier, when you don't need to communicate with multiple ODP instances. 
+With module scoped instance you configure the SDK once and then use it throughout your application.
+
+Creating your own instance of `ODPClient` gives you more control.
+
+### Configuration and usage
 
-You need to configure the SDK with your API keys. If you are only sending data to ODP, you normally only need your public API key, however, some API calls require your private API key. You can obtain these from the **Account Settings > APIs** page in the ODP app.
+You need to configure the SDK with your API keys. 
+If you are only sending data to ODP, you normally only need your public API key, however, some API calls require your private API key. 
+You can obtain these from the **Account Settings > APIs** page in the ODP app.
 
 [block:image]
 {
-  "images": [
-    {
-      "image": [
-        "https://files.readme.io/43ca706-image.png",
-        null,
-        null
-      ],
-      "align": "center",
-      "border": true
-    }
-  ]
+"images": [
+{
+"image": [
+"https://files.readme.io/43ca706-image.png",
+null,
+null
+],
+"align": "center",
+"border": true
+}
+]
 }
 [/block]
 
+
+#### Using module scoped instance
 ```typescript
-import {z} from '@zaiusinc/node-sdk';
-z.configure({
+import {odp} from '@zaiusinc/node-sdk';
+odp.configure({
   apiKey: 'your_public_or_private_api_key'
 });
+
+const event = {type: 'pageview', identifiers: {email}, data: {page}};
+await odp.event(event);
 ```
 
-## Usage
+Alternatively, you can provide the `apiKey` as an environment variable `ODP_SDK_API_KEY` and omit `cofigure` method.
 
-ODP APIs are exposed through the `z` export:
+> **Note** 
+> For compatibility with previous versions of the SDK, module scoped instance is also exported as `z` from the SDK.
 
-```javascript
-import {z} from '@zaiusinc/node-sdk';
+#### Creating your own instance of `ODPClient`
+```typescript
+import {ODPClient} from '@zaiusinc/node-sdk';
+const odp = new ODPClient({
+  apiKey: 'your_public_or_private_api_key'
+});
 
-async function pageview(email, page) {
-  const event = {type: 'pageview', identifiers: {email}, data: {page}};
-  const result = await z.event(event);
-  return result.success;
-}
+const event = {type: 'pageview', identifiers: {email}, data: {page}};
+await odp.event(event);
 ```
 
+Alternatively, you can provide the `apiKey` as an environment variable `ODP_SDK_API_KEY` and use parameterless constructor.
```

Comment:
correct, however, this is for a different use case. I think it still makes sense to repeat it as otherwise it wouldn't be clear if ODP_SDK_API_KEY applies also the use case of using own instance of ODPClient